README.md

```diff
@@ -0,0 +1,26 @@
+# Lissi Connect through NodeJS
+
+This express application presents a working example of a Single Sign-On method through lissi-Connect using `passportJS` and `openid-client` strategy.
+
+## Quickstart
+
+Clone this repository using `git clone https://github.com/lissi-id/lissi-oidc-nodejs-sample.git`
+
+Create and fill out a `.env` file at the root of the cloned repository following this template
+
+    CLIENT_ID=
+    CLIENT_SECRET=
+    PRESENTATION_CONFIG_ID=
+    PORT=
+    ISSUER=
+    CREDENTIAL_DEFINITION_ID=
+
+> Please note that `AUTHORIZATION_URL`, `TOKEN_URL` and `CALLLBACK_URL` fields are required using the `passport-openidconnect` strategy, which we do not recommend as custom parameters are not supported by default. Instead, we would prefer the  [`openid-client`](https://www.passportjs.org/packages/openid-client/) strategy as it offers the possibility to discover the issuer endpoint through web finger URL
```

Comment:
Why highlight something that we do not recommend? I would suggest just outlining one way in this sample. Therefore, I think this note can be removed?